### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by jhnns**
Since the imports-loader is very important to tackle compatibility problems, I've decided to add further explanations to the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,88 @@
...
+
+```javascript
+// ./webpack.config.js
+
+module.exports = {
+    ...
+    module: {
+        loaders: [
+            {
+                test: /[\/\\]node_modules[\/\\]some-module[\/\\]index\.js$/
```
Comment (sokra) [inline README.md]: Can you use an absolute path here? That's easier to read:

``` js
test: path.resolve(__dirname, "node_modules/some-module/index.js")
```

File: README.md [hunk 2]
```diff
@@ -1,17 +1,88 @@
...
+
+### Disable AMD
+
+There are many modules that check for a `define` function before using CommonJS. Since webpack is capable of both, they default to AMD in this case, which can be a problem if the implementation is quirky.
+
+Then you can easily disable the AMD path by writing
+
+```javascript
+imports?define=>false
+```
```
Comment (sokra) [inline README.md]: A link to the "shimming modules" documentation page would be great.

**Comment by jhnns** [comment]
done()

**Comment by jhnns** [comment]
Thx for write access :)

**Pull request merged**